index.html

```diff
@@ -5234,38 +5309,78 @@ <h3>Personally Identifiable Information</h3>
 transit, as well as encryption or data access control mechanisms to protect
 the data in a [=verifiable credential=] while at rest.
         </p>
+
+        <p>
+In general, individuals are advised to assume that a [=verifiable credential=],
+like most physical credentials, will leak personally identifiable information
+when shared. To combat this leakage, the [=verifiable credential=], and the
+securing mechanism, need to be specifically designed to avoid correlation.
+[=Verifiable credentials=] that are specifically designed to prevent the leakage
+of personally identifiable information do exist. Individuals and implementers
+are urged to prefer these types of credentials over ones that are not designed
+to protect personally identifiable information.
+        </p>
```

Comment:
This kind of advice is certainly not normative and arguably belongs in a deployment guide, rather than the specification.  At most, it's non-normative deployment considerations.  Please delete this addition.

Reply:
This change was requested by @jyasskin. Its in a privacy considerations section, which tends to NOT include normative statements but general non-normative deployment or usage considerations.

Please discuss your request to remove this guidance with @jyasskin.